.gitignore

```diff
@@ -1 +1,3 @@
-node_modules/
\ No newline at end of file
+node_modules/
+package-lock.json
+createGoalTest.js
```

Comment:
Why add package-lock.json in gitignore and why test file too?

Reply:
Because the test file is not yet complete

Reply:
reverted